### DISCUSSION TITLE
EMFILE: too many open files when building output

### DISCUSSION THREAD

**Discussion by noxxxxxxxx** [nuxt/nuxt#16625] [Questions]
```shell
Nuxi 3.0.0-rc.12-27759910.9e6d292
Nuxt 3.0.0-rc.12-27759910.9e6d292 with Nitro 0.5.5-27758232.d90c938 

pnpm 7.12.2
v16.16.0 (npm v8.11.0)
```
this error block the build process when running`npm run build`，any solution?

package.json

```json
{
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev --port 3331",
    "generate": "nuxt generate",
    "preview": "nuxt preview"
  },
  "devDependencies": {
    "@babel/eslint-parser": "^7.19.1",
    "@nuxtjs/eslint-config-typescript": "^11.0.0",
    "@types/node": "^18.7.23",
    "@typescript-eslint/eslint-plugin": "^5.38.1",
    "@typescript-eslint/parser": "^5.38.1",
    "eslint": "8.22.0",
    "eslint-plugin-nuxt": "^4.0.0",
    "eslint-plugin-vue": "^9.5.1",
    "nuxt": "npm:nuxt3@latest",
    "nuxt-windicss": "^2.5.2",
    "sass": "^1.54.5",
    "typescript": "^4.8.4",
    "vite-plugin-eslint": "^1.8.1"
  },
  "dependencies": {
    "@icon-park/vue-next": "^1.4.2",
    "vue-toastification": "2.0.0-rc.5"
  }
}

```

![image](https://user-images.githubusercontent.com/37919656/195740531-db176325-43a3-4f01-b3e7-43db8ec09773.png)

**Comment by howbizarre** [comment]
So strange, I've gotten the same error on my Windows 11 laptop, but on my Ubuntu station all is OK, and it couse since the last couple of days. I use my Windows laptop when I go outside, cos' it's small and light, and I do not synchronize the source all the time, just before leaving. The last time was a few days before... and today after getting the repository I saw this error...

<img width="679" alt="image" src="https://github.com/nuxt/nuxt/assets/112457/555b93dd-f5d0-415c-a6f2-dd980904ba2f">

I do not know where it comes from. The one of biggest differences is the Firebase version - from 9 to 10. All others are minors.

My Nuxt Info is:
------------------------------
- Operating System: Windows_NT
- Node Version:     v18.17.0
- Nuxt Version:     3.6.5
- Nitro Version:    2.5.2
- Package Manager:  npm@9.6.7
- Builder:          vite
- User Config:      runtimeConfig, modules, tailwindcss, nitro, app, typescript, routeRules, devtools
- Runtime Modules:  @nuxtjs/tailwindcss@6.8.0, nuxt-icon@0.4.2, @pinia/nuxt@0.4.11
- Build Modules:    -
------------------------------

Has anyone else got a similar error and how to fix it?..

**Comment by howbizarre** [comment]
I found the reason cause the error in my case.
It was a line `content: ["./**/*.{vue,ts}"],` in Tailwind configuration...

```typescript
export default defineNuxtConfig({
  ...
  tailwindcss: {
    ...
    config: {
      content: ["./**/*.{vue,ts}"],
      ...
    },
  },
});
```
 
After removing the line - everything is back to normal under the Windows...
This line comes from the previous code base of the app, where we do not use Nuxt.

Hope to help somebody... 🖖
